tests/test_connectors_salesforce_errors.py

```
@@ -1,23 +1,43 @@
+    ConnectorNotFoundError,
+    ConnectorRateLimitError,
+)
+from siege_utilities.connectors.salesforce import (
+    SOQLQuery,
+    SalesforceConnector,
+)
 
 
 class _FakeResp:
-    def __init__(self, status_code, *, json_body=None, text="", json_raises=False):
+    def __init__(
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)